cypress-io/cypress
# chore: commit packages/graphql/schemas/schema.graphql from PR #29420

- Closes https://github.com/cypress-io/cypress/issues/29656

### Additional details

- PR https://github.com/cypress-io/cypress/pull/29420 leaves change to autogenerated [packages/graphql/schemas/schema.graphql](https://github.com/cypress-io/cypress/blob/develop/packages/graphql/schemas/schema.graphql) uncommitted.

### Steps to test

Execute:

```shell
git clean -xfd
yarn
git status
```

and confirm that there are no differences reported for `packages/graphql/schemas/schema.graphql`

### How has the user experience changed?

This issue affects contributors only.

### PR Tasks

- [na] Have tests been added/updated?
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)?
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
